bin/migrate-split-repo.sh

```diff
@@ -41,3 +41,6 @@ ${SCRIPT_DIR}/migrate-git-history.sh \
   "${PACKAGE_PATH}" \
```

Comment:
It looks like this file's mode is changing to not be executable?

Reply:
Fixed it, thanks!